tree/ntuple/v7/inc/ROOT/RNTupleDescriptor.hxx

```diff
@@ -226,7 +226,7 @@ public:
    /// The window of element indexes of a particular column in a particular cluster
    struct RColumnRange {
       DescriptorId_t fPhysicalColumnId = kInvalidDescriptorId;
-      /// A 64bit element index
+      /// The global index of the first column element in the cluster
```

Comment:
Consider
```suggestion
      /// The global index of the first element of (all) the columns in the cluster
```

Reply:
Is this the case though? The first element of the payload column of a vector field will most likely have a different global index than its offset column.